packages/create-workleap/jest.config.ts

```diff
@@ -1,11 +1,11 @@
-import type {Config} from 'jest';
+import type { Config } from "jest";
 
 const config: Config = {
   testEnvironment: "node",
-  testRegex: "/tests/*/.*\\.test\\.ts$",
+  testRegex: "/test/*/.*\\.test\\.ts$",
```

Comment:
Is there a specific reason why you removed the "s" from "tests" ?

There seems to be many tests files.